pkg/kubelet/kubelet.go

```diff
@@ -858,7 +858,7 @@ func (kl *Kubelet) killContainerByID(ID string) error {
 }
 
 const (
-	PodInfraContainerImage = "kubernetes/pause:latest"
+	PodInfraContainerImage = "gcr.io/google_containers/pause:0.8.0"
```

Comment:
Have we updated the build script/makefile so we push this to gcr.io when updated? Also, I haven't seen the 0.8.0 tag, is that one you pushed?